src/commands/broadcast.ts

```diff
@@ -230,24 +230,25 @@ export const broadcastEventsCommand = async (ctx: CommandContext<Context>) => {
   }
   
   try {
-    const events = await DrizzleDatabaseService.getAllUpcomingEvents();
-    
-    if (events.length === 0) {
+    const allEvents = await DrizzleDatabaseService.getAllUpcomingEvents();
+    const sortedEvents = sortEventsByDate(allEvents);
+
+    if (sortedEvents.length === 0) {
       await ctx.reply('❌ No events to broadcast.');
       return;
     }
     
     let broadcastMessage = '📅 *Upcoming Events*\n\n';
     
-    events.forEach(event => {
+    sortedEvents.forEach(event => {
       const eventDate = new Date(event.date).toLocaleDateString();
       const escapedTitle = escapeMarkdown(event.title);
       const escapedVenue = escapeMarkdown(event.venue || 'TBD');
       const escapedFormat = escapeMarkdown(event.format);
       
       broadcastMessage += `*${escapedTitle}*\n`;
       broadcastMessage += `📍 ${escapedVenue}\n`;
-      broadcastMessage += `📅 ${eventDate}\n`;
+      broadcastMessage += `Date: ${formatHumanDate(eventDate)}\n`;
```

Comment:
Calendar emoji should still remain

```suggestion
      broadcastMessage += `📅 ${formatHumanDate(eventDate)}\n`;
```